content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Production Readiness - Shared Cloud"
```

Comment:
Let's just call this section "Production Readiness"

Reply:
Updated through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a